### DISCUSSION TITLE
Run Monitor Job on all Nodes in Cluster?

### DISCUSSION THREAD

**Discussion by stlawrence** [quartznet/quartznet#1657] [Q&A] [answered]
With UseClustering() is there a way to have a monitoring job that runs on all nodes instead of one randomly selected node? Or some way to override the node selection to implement a round robin approach so a job scheduled for once a minute would run on all 3 nodes every 3 minutes?

Use case is a job that monitors for cancelled jobs in order to implement Interrupt across the cluster. 
(I'm also be open to other ways to achieve this goal.)

**Comment by lahma** [comment, marked as answer]
There's no feature for pinning or choosing node per run. Might it be easier to add a second scheduler to each node that would run the job in fixed intervals based on node number (in-memory, no clustering)?

**Reply by stlawrence** [reply]
Thanks! I was looking to do this using System.Threading.Timer but your suggestion of using a second InMemory scheduler was much easier.
